x-pack/plugins/observability_shared/public/components/page_template/search_bar_portal.tsx

```diff
@@ -7,10 +7,10 @@
 
 import React, { ReactNode, useEffect, useMemo } from 'react';
 import ReactDOM from 'react-dom';
-import { EuiPanel } from '@elastic/eui';
+import { EuiPanel, EuiSpacer } from '@elastic/eui';
 import { createHtmlPortalNode, InPortal, OutPortal } from 'react-reverse-portal';
 export interface Props {
-  children: ReactNode;
+  children?: ReactNode;
```

Comment:
Why did you make children optional here?

Reply:
i think it just some bad conflicts resolution, i will revert